pkg/webhook/mutation/pod/metadata/workload_test.go

```diff
@@ -90,31 +90,37 @@ func TestFindRootOwnerOfPod(t *testing.T) {
 		assert.Equal(t, "Pod", workloadInfo.kind)
 	})
 
-	t.Run("should be empty if owner is not well known", func(t *testing.T) {
+	t.Run("should be pod if owner is not well known", func(t *testing.T) {
 		pod := corev1.Pod{
 			TypeMeta: metav1.TypeMeta{
 				Kind: "Pod",
 			},
 			ObjectMeta: metav1.ObjectMeta{
 				OwnerReferences: []metav1.OwnerReference{
 					{
-						APIVersion: "unknown",
-						Kind:       "unknown",
+						APIVersion: "v1",
+						Kind:       "Secret",
```

Comment:
wouldn't `unknown` be fine here? It should just be a resource that is not well known